cmd/organization/list/display_table.go

```diff
@@ -20,15 +20,20 @@ var (
 	rowStyle = lipgloss.NewStyle().Padding(0, 1)
 )
 
-func setupTable(organizations []organization.OrganizationMembership) *table.Table {
-	columns := []string{"Name", "Slug", "Role", "ID"}
+func setupTable(organizations []organization.OrganizationMembership, configuredOrganizationSlug string) *table.Table {
+	columns := []string{"Name", "Slug", "Role", "ID", "Default"}
 	var rows [][]string
 	for _, o := range organizations {
+		isDefault := ""
+		if o.Organization.Slug == configuredOrganizationSlug {
+			isDefault = "*"
```

Comment:
I don't think that's worth it to be honest. Like you ask in a different comment, we changed the default display mode to list, since tabular display is problematic in a terminal setting.